src/components/DashboardSideNav.tsx

```diff
@@ -1,9 +1,8 @@
 import { Triangle } from 'lucide-react';
 import { ReactNode, useState } from 'react';
-import { appLogo } from '../utils/images';
 import Button from './buttons/Button';
+import { appLogo } from '../utils/images';
 import { dashProfileIcon } from '../utils/images';
-
 const DashboardSideNav = ({
 	children,
 	otherStyles,
```

Comment:
inside this dashboard sideNva I need to the username to be dynamic and same case for profile picture.

Reply:
on this we issue we proposed that we will use user profile to update the picture and username so we have to wait for  michelle's task to be merged and prepare the fix for it  @leandreAlly